sl/CMakeLists.txt

```diff
@@ -83,8 +83,8 @@ add_library(predator STATIC
 # build compiler plug-in (libsl.so/.dylib)
 CL_BUILD_COMPILER_PLUGIN(sl predator ../cl_build)
 
-# get the full path of libsl.so/.dylib
-get_property(SL_PLUG TARGET sl PROPERTY LOCATION)
+# set the full path of libsl.so/.dylib
+set(SL_PLUG ${cl_BINARY_DIR}/tests/libsl${CMAKE_SHARED_LIBRARY_SUFFIX})
```

Comment:
This does not look correct.  Why would we put the `sl` plug-in into `${cl_BINARY_DIR}/tests`?